soc/riscv/openisa_rv32m1/Kconfig.defconfig

```diff
@@ -122,7 +122,7 @@ config RV32M1_INTMUX_CHANNEL_7
 
 endif # MULTI_LEVEL_INTERRUPTS
 
-config PINMUX_RV32M1
+config PINCTRL_RV32M1
 	default y
```

Comment:
this seems redundant, driver defaults to `y` if dt compatible is enabled

Reply:
Removed it altogether.